sunpy/map/tests/test_mapbase.py

```diff
@@ -941,9 +941,9 @@ def test_rotate(aia171_test_map):
     assert aia171_test_map_crop_rot.data.shape[0] < aia171_test_map_crop_rot.data.shape[1]
 
 
-@pytest.mark.xfail(version.parse(np.__version__) >= version.parse("1.2.0"),
-                   reason="Numpy >= 1.20.0 doesn't allow NaN to int conversion")
-def test_rotate_with_incompatible_missing_dtype():
+@pytest.mark.skipif(version.parse(np.__version__) >= version.parse("1.20.0"),
```

Comment:
```suggestion
@pytest.mark.xfail(version.parse(np.__version__) >= version.parse("1.20.0"),
```
I think we should check this fails instead of just skipping? Similarly for the new test below.

Reply:
Good point.